spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -259,9 +259,9 @@ def do_signin(email, password)
           msg = "Attempted signup from suspected spammer, " \
                 "email: spammer@example.com, " \
                 "name: 'Download New Person 1080p!'"
-          expect(Rails.logger).to receive(:info).with(msg)
-
+          allow(Rails.logger).to receive(:info)
           do_signin(user.email, 'password1234')
+          expect(Rails.logger).to have_received(:info).with(msg)
```

Comment:
Would be good to include the failing spec output in the commit message, as its really difficult to understand _why_ this was failing.

Seems fine in principle though.

Reply:
turns out, it was the `allow` that was the magic bit - fixed in https://github.com/mysociety/alaveteli/pull/5098/commits/14a1c17af6fa7f5e823c310f24434d9ca2a5053f